src/app/Components/Results/AddressResults/index.jsx

```diff
@@ -95,18 +95,32 @@ export default class AddressResults extends Component {
                   </td>
 
                   <td className="govuk-table__cell">
-                    {this.props.tenancies[i].residents.map(resident => (
-                      <p>{resident.firstName + ' ' + resident.lastName}</p>
-                    ))}
+                    {this.props.tenancies[i].residents.map(
+                      resident =>
+                        resident.responsible && (
+                          <p>
+                            {resident.title +
+                              ' ' +
+                              resident.firstName +
+                              ' ' +
+                              resident.lastName}
```

Comment:
This would look a lot better in string concatenation:
`'${resident.title} ${resident.firstName} ${resident.lastName}'`